src/plugin/node.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * v11.1.0
+ * Additional ESLint rules for Node.js
+ *
+ * https://github.com/mysticatea/eslint-plugin-node
+ */
+ module.exports = {
+    plugins: ['node'],
+    rules: {
```

Comment:
which of these, if any, are new? or were they just moved from another file

Reply:
There were quite a few new ones, so I've grouped them together for now so it's easier to review. All new rules are from line 22 onward.